hazelcast/src/test/java/example/serialization/InnerDTO.java

```diff
@@ -55,7 +54,7 @@
     }
 
     @SuppressWarnings("checkstyle:ParameterNumber")
-    public InnerDTO(boolean[] bools, byte[] bb, char[] cc, short[] ss, int[] ii, long[] ll, float[] ff, double[] dd, NamedDTO[] nn,
+    public InnerDTO(boolean[] bools, byte[] bb, short[] ss, int[] ii, long[] ll, float[] ff, double[] dd, NamedDTO[] nn,
```

Comment:
This class is also used from portable test. If we remove char then Portable will have less testing. Should we have two classes, `PortableInnerDTO` and `CompactInnerDTO`?

Reply:
Can you share the portable test that uses this? I could only find references to this in the compact tests

Reply:
Ok, that was a false alarm. Seems like only used from compact. It would be great if all compact specific such class would be under some package like `test.compact`.